internal/scorer/scorer.go

```diff
@@ -103,10 +83,19 @@ func NameFromFilepath(filepath string) string {
 	// Get the name of the file used, without the path
 	f := path.Base(filepath)
 	ext := path.Ext(f)
-	// Strip the extension and convert to lowercase
-	f = strings.ToLower(strings.TrimSuffix(f, ext))
-	// Change any non-alphanumeric character into an underscore
-	f = regexp.MustCompile("[^a-z0-9_]").ReplaceAllString(f, "_")
+
+	modified := func(r rune) rune {
```

Comment:
Consider using `modifier` for the function name.

Reply:
I have changed `modified` to `modifier`.